test/test_linalg.py

```diff
@@ -19,6 +19,44 @@
 class TestLinalg(TestCase):
     exact_dtype = True
 
+    @dtypes(torch.float, torch.double)
```

Comment:
torch.double variant is probably not very interesting; I think this can just test torch.float?